src/main/java/ladder/view/InputView.java

```diff
@@ -0,0 +1,69 @@
+package ladder.view;
+
+import ladder.constant.MessageConstant;
+import ladder.model.LadderGoal;
+import ladder.model.LadderPlayer;
+import ladder.validator.LadderGoalValidator;
+import ladder.validator.LadderHeightValidator;
+import ladder.validator.LadderPlayerValidator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class InputView {
+    private static final String NEW_LINE = "\n";
+    private static final String DELIMITER = ",";
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public static List<LadderPlayer> createLadderPlayers() {
+        System.out.println(MessageConstant.INPUT_LADDER_PLAYER_NAME);
+        return createLadderPlayers(SCANNER.nextLine().split(DELIMITER));
+    }
+
+    public static List<LadderPlayer> createLadderPlayers(String[] inputs) {
+        try {
+            LadderPlayerValidator.checkAccuracyOfUserInputs(inputs);
+            return Arrays.stream(inputs).map(String::trim).map(LadderPlayer::new).collect(Collectors.toList());
```

Comment:
스트림 메서드 호출은 한줄에 하나씩 하면 가독성이 좀 더 좋아집니다 :)
```
return Arrays.stream(inputs)
                .map(String::trim)
                .map(LadderPlayer::new)
                .collect(Collectors.toList());
```

출처 : https://marcin-chwedczuk.github.io/java-streams-best-practices
> You should have at most one stream method call per line. 
> This will make stream operations like map, filter and  collect easily recognizable.
```